### PR TITLE
Ng private data

### PR DESCRIPTION
**Pull request by mhevery**

### CODE REVIEW

**Review by kara — APPROVED** [review]
LGTM in general, though CI says there's a hello-world size regression. Seems like we should look into that before merging?

**Comment by mhevery** [comment]
I think size error is some kind of flake because the sizes of bundles here https://circleci.com/gh/angular/angular/326515#artifacts/containers/0 are correct if you download them.

**Comment by mhevery** [comment]
http://test/OCL:248834900:BASE:248834905:1558155226853:8842a15a

File: packages/core/test/linker/ng_module_integration_spec.ts [hunk 1]
```diff
@@ -142,7 +142,7 @@ function declareTests(config?: {useJit: boolean}) {
         // may face a problem where previously compiled defs available to a given
         // Component/Directive are cached in TView and may become stale (in case any of these defs
         // gets recompiled). In order to avoid this problem, we force fresh TView to be created.
-        componentDef.template.ngPrivateData = null;
+        componentDef.TView = null;
```
Comment (JoostK) [inline packages/core/test/linker/ng_module_integration_spec.ts]: I am surprised this typo (`TView` -> `tView`) does not cause a failure on CI. Is this test not ran under Bazel perhaps?

**Comment by mhevery** [comment]
CARETAKER: The size regression is unrelated and has since been fixed by @ocombe. Ready to merge

**Review by kara — APPROVED** [review]

**Pull request closed without merging**